src/blocks/scratch3_sensing.js

```diff
@@ -143,6 +143,7 @@ class Scratch3SensingBlocks {
         ));
 
         if (currentlyAsking) {
+            this.runtime.emit('SAY', stopTarget, 'say', null);
```

Comment:
This line causes the error below and the say bubble does not actually go away. The `null` should be `''` instead.

```
Uncaught TypeError: Cannot read property 'normalize' of null
    at SVGTextWrapper.wrapText (text-wrapper.js:40)
    at SVGTextWrapper.wrapText (svg-text-wrapper.js:114)
    at SVGTextBubble._buildTextFragment (svg-text-bubble.js:178)
```

Reply:
```suggestion
            this.runtime.emit('SAY', stopTarget, 'say', '');
```